packages/ek-components/src/components/EkPlayButton.vue

```diff
@@ -105,4 +113,9 @@ export default {
   text-transform: uppercase;
 }
 
+button:disabled {
+  opacity: 0.33;
+  background-color: rgba(0, 0, 0, 0.25);
```

Comment:
Two options for de-hardcoding these values:
- Add new variables to the theme at `packages/ek-components/src/styles.scss` (if this is wanted only for EkPlayButton)
- Change the style of all disabled buttons in the theme at `packages/ek-components/src/index.scss` (if this is wanted for all disabled buttons)

Reply:
Done!